### PR TITLE
add disabled as prop to support visibilty modes of tooltip

### PR DESCRIPTION
**Pull request by sbatel**
## What this PR does 
this change is needed to prevent situations like this - 
image
we would like to hide the tooltip when the popover is open (the tooltip wraps the button the is wrapped by the popover)

Support the ability to disable/enable tooltip (control visibility).
similar to bootstrap's tooltip disable prop -> https://getbootstrap.com/docs/4.0/components/tooltips/#tooltipdisable

## Screenshots & Recordings 

https://github.com/vimeo/iris/assets/38322603/c5e3b41d-fca1-49c6-aa49-6f49c77fa36b

## How it does that 
Added the 'disabled' prop (default value is false) that effects the Tip's style (visibility: visible/hidden)

In addition I edited the Active story to reflect better what exactly it does.
 

https://github.com/vimeo/iris/assets/38322603/cd42e954-0738-4032-b41e-d84040228113

 ## Tests
Add unit tests to the component on file Tip.test.tsx

### CODE REVIEW

**Comment by juliewongbandue** [comment]
Hey @sbatel can you add unit tests for this component as well?

File: src/components/Tip/Tip.types.ts [hunk 1]
```diff
@@ -8,6 +8,11 @@ export type Props = IrisProps<{
    * If not defined, it will use the internal state.
    */
   active?: boolean;
+  /**
+   * Controlls the tooltip visibility.
```
Comment (juliewongbandue) [inline src/components/Tip/Tip.types.ts:12]: Small spelling nit:

```suggestion
   * Controls the tooltip visibility.
```

**Review by juliewongbandue — CHANGES_REQUESTED** [review]
Just a small nit!

File: src/components/Tip/Tip.types.ts (hunk 1 above)
Comment (sbatel) [reply src/components/Tip/Tip.types.ts]: :) fixed!

**Review by juliewongbandue — APPROVED** [review]

**Pull request merged**